attestation-agent/attester/src/csv/mod.rs

```diff
@@ -30,8 +31,9 @@ struct CertificateChain {
 
 #[derive(Serialize, Deserialize)]
 struct CsvEvidence {
-    attestation_report: AttestationReport,
+    attestation_report: String,
     cert_chain: CertificateChain,
+    serial_number: Vec<u8>,
```

Comment:
Could you provide more context about this PR? Change `attestation_report` to `String` may help to present the whole evidence as string, but `serial_number` is still `Vec<u8>`